amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -9,22 +9,27 @@ import akka.util.ByteString
 import com.rabbitmq.client.AMQP.BasicProperties
 import com.rabbitmq.client._
 
-final case class OutgoingMessage(bytes: ByteString, immediate: Boolean, mandatory: Boolean, props: Option[BasicProperties])
+final case class OutgoingMessage(bytes: ByteString,
+                                 immediate: Boolean,
+                                 mandatory: Boolean,
+                                 props: Option[BasicProperties])
```

Comment:
(I don't mind it as it is terribly much, the comment was not to object against merging, more a reflection)